.github/ISSUE_TEMPLATE/icon_request.yml

```
@@ -17,8 +17,9 @@ body:
         you're requesting then leave a comment on it or add a reaction.
 
         > [!TIP]
-        > Similarweb is now trying to force users to log in in order to view statistics. You can bypass this by going directly to:
-        > `https://similarweb.com/website/google.com`, replacing `google.com` with the TLD you would like to get the stats on!
+        > In order to view website rankings on `similarweb.com`:
+        > Go to Products at the navbar on top, click on Free tools and then select Analyze a website.
+        > Or go to `https://similarweb.com/website`, enter the website address and click on Analyze Traffic.
```

```suggestion
        > Go to `https://similarweb.com/website`, enter the website address, and click on Analyze Traffic.
```

Is the first method necessary to include?